### PR TITLE
Update secured_private_asset_transfer_tutorial.md

### PR DESCRIPTION
**Pull request by amar141989-dev**
Updated the document to fill the gaps in commands inputs

#### Type of change

- Documentation update

#### Description

There were some gaps in the commands specific in the ReadMe file. Have updated those commands and working as per the underlined code output. 

#### Additional details

- Added environment variable ASSET_ID to store and use asset id created by fabric 
- Changed command payload for AgreeToBuy command 
- Corrected output of some commands

### CODE REVIEW

File: docs/source/secured_asset_transfer/secured_private_asset_transfer_tutorial.md [hunk 1]
```diff
@@ -13,19 +13,19 @@ The private asset transfer scenario is bound by the following requirements:
 
 - An asset may be issued by the first owner's organization (in the real world issuance may be restricted to some authority that certifies an asset's properties).
 - Ownership is managed at the organization level (the Fabric permissioning scheme would equally support ownership at an individual identity level within an organization).
-- The asset identifier is a hash of the asset's immutable properties, and along with the current owner is stored as public channel data for all channel members to see.
```
Comment (denyeart) [inline docs/source/secured_asset_transfer/secured_private_asset_transfer_tutorial.md]: Thanks for the update.

Question - why are you removing the information about the asset hash throughout this document? It was intentionally documented as a concept for users to understand.

Here is the corresponding code that creates the hash:
https://github.com/hyperledger/fabric-samples/blob/main/asset-transfer-secured-agreement/chaincode-go/asset_transfer.go#L59-L62

Comment (amar141989-dev) [reply docs/source/secured_asset_transfer/secured_private_asset_transfer_tutorial.md]: Thanks for pointing this out. I've restored the descriptions as my changes were more towards the command's inputs and their outputs.

Comment (amar141989-dev) [reply docs/source/secured_asset_transfer/secured_private_asset_transfer_tutorial.md]: Looks like it is happening dur to the way type is declared in  https://github.com/hyperledger/fabric-samples/blob/main/asset-transfer-secured-agreement/chaincode-go/asset_transfer.go

**Review by denyeart — APPROVED** [review]
With the latest updates this PR only changes the output fields from using underscores to camelCase.

I'm fine with that and will merge, but if you intended to do more go ahead and open another pull request.

**Pull request merged**